.github/workflows/action-test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: "ubuntu-latest"
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7.10, 3.8, 3.9]
```

Comment:
Any reason for using this very specific pin? Wouldn't just 3.7 always get you the latest minor release?